tests/baselines/reference/controlFlowWhileStatement.errors.txt

```diff
@@ -0,0 +1,113 @@
+tests/cases/conformance/controlFlow/controlFlowWhileStatement.ts(81,12): error TS2365: Operator '>' cannot be applied to types 'string | number' and 'number'.

```

Comment:
Perhaps this indicated that the improvement requires some more work - or maybe that it's not a good improvement after all, as maybe "swallowing" non-numbers in unions is desired? Kinda easy to miss a mistake because of this behavior but some existing code can depend on it